File: Java/Maths/Area of Rhombus.java
import java.util.Scanner;
class rhombus {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the diagonal 1: ");
        double p = input.nextDouble();
        System.out.print("Enter the diagonal 2: ");
        double q = input.nextDouble();

        double area = (p * q) / 2; // or, area = 0.5 * (p * q)
        System.out.println("The area of rhombus is = " +  area);
    }
}

File: C++/Maths/Area of Rhombus.cpp
#include <iostream>
using namespace std;

int main(){
    double p, q;
    cout << "Enter the diagonal 1: ";
    cin >> p;
    cout << "Enter the diagonal 2: ";
    cin >> q;

    double area = (p * q) / 2; // or, area = 0.5 * (p * q)
    cout << "The area of rhombus is = " << area;
}